pom.xml

```diff
@@ -80,6 +80,8 @@
 
     <jboss-log-manager.version>2.1.17.Final</jboss-log-manager.version>
 
+    <opentelemetry.version>0.6.0</opentelemetry.version>
```

Comment:
Could we bump it to the latest version 0.7.0?

Reply:
@kenfinnigan I believe this was done, no?

Reply:
Yes